scripts/scratch/steps/0160-deploy-dual-governance.ts

```diff
@@ -0,0 +1,376 @@
+  const dgDotEnvFilePath = `${DG_INSTALL_DIR}/.env`;
+
+  return writeFile(fileContent, dgDotEnvFilePath, ".env");
+}
+
+async function writeFile(fileContent: string, filePath: string, fileKind: string) {
+  try {
+    await fs.writeFile(filePath, fileContent, "utf8");
+    log.success(`${fileKind} file successfully saved to ${filePath}`);
+  } catch (error) {
+    log.error(`An error has occurred while writing DG ${filePath} file`, `${error}`);
+    throw error;
+  }
+}
+
+async function getLatestDGDeployArtifactFilename(networkChainId: string) {
+  const deployArtifactFilenameRe = new RegExp(`deploy-artifact-${networkChainId}-\\d+.toml`, "ig");
+
+  let files = [];
+  try {
+    files = await fs.readdir(DG_DEPLOY_ARTIFACTS_DIR);
+  } catch (error) {
+    log.error("An error has occurred while reading directory:", `${error}`);
+    throw error;
+  }
+
+  files = files.filter((file) => file.match(deployArtifactFilenameRe)).sort();
+
+  if (files.length === 0) {
+    throw new Error("No deploy artifact file found");
+  }
+
+  return files[files.length - 1];
+}
+
+function getDGConfig(chainId: string, networkState: DeploymentState) {
+  const daoVoting = getAddress(Sk.appVoting, networkState);
+  const withdrawalQueue = getAddress(Sk.withdrawalQueueERC721, networkState);
+  const stEth = getAddress(Sk.appLido, networkState);
+  const wstEth = getAddress(Sk.wstETH, networkState);
+
+  if (!networkState[Sk.dualGovernanceConfig]) {
+    throw new Error("DG deploy config is not set, please specify it in the deployed-testnet-defaults.json file");
+  }
+
+  return {
+    chain_id: chainId,
+    dual_governance: {
+      admin_proposer: daoVoting,
+      proposals_canceller: daoVoting,
+      sealable_withdrawal_blockers: [], // TODO: add withdrawalQueue
+      reseal_committee: daoVoting,
+      tiebreaker_activation_timeout:
+        networkState[Sk.dualGovernanceConfig].dual_governance.tiebreaker_activation_timeout,
+
+      signalling_tokens: {
+        st_eth: stEth,
+        wst_eth: wstEth,
+        withdrawal_queue: withdrawalQueue,
+      },
+      sanity_check_params: {
+        max_min_assets_lock_duration:
+          networkState[Sk.dualGovernanceConfig].dual_governance.sanity_check_params.max_min_assets_lock_duration,
+        max_sealable_withdrawal_blockers_count:
+          networkState[Sk.dualGovernanceConfig].dual_governance.sanity_check_params
+            .max_sealable_withdrawal_blockers_count,
+        max_tiebreaker_activation_timeout:
+          networkState[Sk.dualGovernanceConfig].dual_governance.sanity_check_params.max_tiebreaker_activation_timeout,
+        min_tiebreaker_activation_timeout:
+          networkState[Sk.dualGovernanceConfig].dual_governance.sanity_check_params.min_tiebreaker_activation_timeout,
+        min_withdrawals_batch_size:
+          networkState[Sk.dualGovernanceConfig].dual_governance.sanity_check_params.min_withdrawals_batch_size,
+      },
+    },
+    dual_governance_config_provider: {
```

Comment:
Just
```suggestion
    dual_governance_config_provider: networkState[Sk.dualGovernanceConfig].dual_governance_config_provider
```
here and similar places, why not?